packages/cardano-services-client/package.json

```diff
@@ -39,6 +39,7 @@
     "module-fixup": "shx cp ../../build/cjs-package.json ./dist/cjs/package.json && cp ../../build/esm-package.json ./dist/esm/package.json",
     "tscNoEmit": "shx echo typescript --noEmit command not implemented yet",
     "cleanup:dist": "shx rm -rf dist",
+    "circular-deps:check": "npx madge --circular dist",
```

Comment:
Any reason we can't install as a `devDependency`?

Reply:
No reason, it's now added as dev dep: bed9d46206e197323ece359fbda5ce82bf4f0f0e